packages/nouns-contracts/contracts/client-incentives/Rewards.sol

```diff
@@ -542,6 +601,15 @@ contract Rewards is NounsClientToken, UUPSUpgradeable, PausableUpgradeable {
      * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
      */
 
+    /**
+     * @dev Only `owner` can call this function
```

Comment:
@eladmallel worth adding a `@notice` explaining what's the significance of this flag

Reply:
added